src/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -48,13 +48,39 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = "Reads a SAM or BAM file and writes a file containing metrics about " +
-                "the statistical distribution of insert size (excluding duplicates) " +
-                "and generates a Histogram plot.",
-        usageShort = "Writes insert size distribution metrics for a SAM or BAM file",
+        usage = CollectInsertSizeMetrics.USAGE_SUMMARY,
+        usageShort = CollectInsertSizeMetrics.USAGE_BRIEF,
         programGroup = Metrics.class
 )
 public class CollectInsertSizeMetrics extends SinglePassSamProgram {
+    static final String USAGE_BRIEF = "Metrics about the insert size distribution of a paired-end library";
+    static final String USAGE_SUMMARY = "Metrics about the insert size distribution of a paired-end library, created by the " +
+            "CollectInsertSizeMetrics program and usually written to a file with the extension " +
+            "\".insert_size_metrics.txt\".  In addition the insert size distribution is plotted to a file with the " +
+            "extension \".insert_size_Histogram.pdf\".<br /><br />" +
+            "There are two separate outputs from this tool including distributions of insert sizes as well as " +
+            "grouped information about read pairs and their orientations.  Currently, the tool groups the read " +
+            "pairs into three categories: forward reverse (FR), which are paired-end reads; reverse forward (RF)," +
+            " which are mate pair reads; and reverse-reverse/forward-forward (tandem reads). In general, paired" +
```

Comment:
Are you sure about this distinction between paired-end reads and mate-pair reads? I always thought that mate pairs were what you get when you do paired-end sequencing. If you're right it would probably be worthwhile to include a note to specifically call out this difference, because I'm sure this will confuse a lot of people.